src/main/java/org/broadinstitute/consent/http/resources/TDRResource.java

```diff
@@ -64,4 +81,60 @@ public Response getDatasetByIdentifier(@Auth AuthUser authUser, @PathParam("iden
             return createExceptionResponse(e);
         }
     }
+
+    @POST
+    @Consumes("application/json")
+    @Produces("application/json")
+    @Path("/dar/draft")
+    @RolesAllowed({ADMIN})
+    public Response createDraftDataAccessRequest(
+        @Auth AuthUser authUser,
+        @Context UriInfo info,
+        @QueryParam("identifiers") String identifiers,
+        @QueryParam("projectTitle") String projectTitle) {
+      try {
+        // Ensure that the user is a registered DUOS and SAM user
+        User user = findOrCreateUser(authUser);
+        if (Objects.isNull(identifiers) || identifiers.isBlank()) {
+          throw new BadRequestException("No dataset identifiers were provided");
+        } else {
+          List<Integer> datasetIds = Arrays.asList(identifiers.split(","))
+              .stream()
+              .filter(identifier -> !identifier.isBlank())
+              .map(identifier -> this.datasetService.findDatasetByIdentifier(identifier).getDataSetId())
+              .collect(Collectors.toList());
+        if (datasetIds.isEmpty()) {
+            throw new BadRequestException("No datasets associated with identifiers: " + identifiers);
+        }
+        DataAccessRequest newDar = new DataAccessRequest();
+        DataAccessRequestData data = new DataAccessRequestData();
+        String referenceId = UUID.randomUUID().toString();
+        newDar.setReferenceId(referenceId);
+        data.setReferenceId(referenceId);
+        data.setProjectTitle(projectTitle);
+        newDar.setData(data);
+        newDar.setDatasetIds(datasetIds);
+        DataAccessRequest result = this.darService.insertDraftDataAccessRequest(user, newDar);
+        URI uri = info.getRequestUriBuilder().path("/" + result.getReferenceId()).build();
+        return Response.created(uri).entity(result.convertToSimplifiedDar()).build();
+        }
+      } catch (Exception e) {
+          return createExceptionResponse(e);
+      }
+    }
+
+    // should this be private? otherwise can't test
+    User findOrCreateUser(AuthUser authUser) {
+        // Ensure that the user is a registered DUOS and SAM user
+        User tdrUser;
+        try {
+            tdrUser = this.userService.findUserByEmail(authUser.getEmail());
+        } catch (NotFoundException nfe) {
+            User newTdrUser = new User();
+            newTdrUser.setEmail(authUser.getEmail());
+            newTdrUser.setDisplayName(authUser.getName());
+            tdrUser = this.userService.createUser(newTdrUser);
```

Comment:
This method doesn't actually register them in Sam, that happens in `SamService.postRegistrationInfo(AuthUser authUser)`. You can see how we do this in `SamResource.postSelfTos(@Auth AuthUser authUser)`.

Reply:
Thinking more about this. It might be time to consolidate the create user/register in sam paths. Not for this PR, but for a cleanup story.

Reply:
Similar comment to above, avoid `this` unless necessary.

Reply:
Updated this method which I moved to UserService to call SamDAO.postRegistrationInfo. Does that endpoint actually create the SAM user, too? I didn't see any "create" methods in the Sam classes. Created a stub ticket for the cleanup work: [DUOS-2097](https://broadworkbench.atlassian.net/browse/DUOS-2097).

Reply:
> Does that endpoint actually create the Sam user, too?

Yes, that sam endpoint will create a Sam user. A user in Sam is visible to all Terra services. More details available in Sam itself: https://sam.dsde-prod.broadinstitute.org/#/Users/createUserV2

The urls we use in Consent to send to Sam are in `ServicesConfiguration`. This is the one we use for the endpoint under discussion:

```
  public String postRegisterUserV2SelfUrl() {
    return getSamUrl() + "register/user/v2/self";
  }
```